style.css

```diff
@@ -2646,6 +2621,38 @@ div.comment:first-of-type {
 /*	13. Site Pagination
 /* --------------------------------------------------------------------------------------------- */
 
+.navigation.pagination {
+	padding-top: 3rem;
+}
+
+.pagination .nav-links {
+	display: flex;
+	flex-wrap: wrap;
+	font-size: 1.8rem;
+	font-weight: 600;
+	margin: -2.5rem 0 0 -2.5rem;
+}
+
+.nav-links > * {
+	margin: 2.5rem 0 0 2.5rem;
+}
+
+.pagination a {
+	text-decoration: none;
+}
+
+.pagination a:hover {
+	text-decoration: underline;
+}
+
+.pagination .dots {
+	margin-top: 1.1em;
```

Comment:
Changing this to `transform: translateY( -.3em );` should put it back in the center of the line. Other than that, the pagination looks good to me. :)

Reply:
Is this `em`s or should it be `rem`s?